src/main/java/Wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,13 @@
+package Wolox.training.repositories;
+
+import Wolox.training.models.Book;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+public interface BookRepository extends JpaRepository<Book, Long> {
+
+    @Query("SELECT b FROM Book b WHERE LOWER(b.author) = LOWER(:author)")
```

Comment:
this is not necessary, you only need `Optional<Book> findByAuthor(String author);`
you can read more here https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#jpa.query-methods.query-creation